File: ios/Classes/Enums/Method Channel/Card Merchant/Common/Method Key/MethodChannelCardMerchantMethodKey.swift
//
//  MethodChannelCardMerchantMethodKey.swift
//  Runner
//
//  Created by Ray Chow on 22/4/2022.
//

import Foundation

enum MethodChannelCardMerchantMethodKey: String {
    case tokeniseWithMpgs = "tokenise_with_mpgs",
         createPaymentMethodWithStripe = "create_payment_method_with_stripe",
         tokeniseWithStripe = "tokenise_with_stripe",
         tokeniseApplePayWithStripe = "tokenise_apple_pay_with_stripe",
         tokeniseGooglePayWithStripe = "tokenise_google_pay_with_stripe",
         setupApplePay = "setup_apple_pay",
         showMpgs3ds = "show_mpgs_3ds",
         showStripe3ds = "show_stripe_3d_secure"
}

File: android/src/main/kotlin/com/keychainpay/keychainpay_mpgs/enums/method_channel/card_merchant/common/method_key/MethodChannelCardMerchantMethodKey.kt
package com.keychainpay.keychainpay_mpgs.enums.method_channel.card_merchant.common.method_key

enum class MethodChannelCardMerchantMethodKey(val key: String) {
    TOKENISE_WITH_MPGS("tokenise_with_mpgs"),
    CREATE_PAYMENT_METHOD_WITH_STRIPE("create_payment_method_with_stripe"),
    TOKENISE_WITH_STRIPE("tokenise_with_stripe"),
    SHOW_MPGS_3DS("show_mpgs_3ds"),
    SHOW_STRIPE_3D_SECURE("show_stripe_3d_secure")
}